### PR TITLE
Configure coverity scans to run for coverity_scan branch

### PR DESCRIPTION
**Pull request by amshinde**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,21 @@ compiler:
...
+   - coverity_scan_run_condition='"$CC" = gcc'
+
+script: ./autogen.sh && make
+
+addons:
+  coverity_scan:
+    project:
+      name: "clearcontainers/shim"
+      description: "Build submitted via Travis CI"
+    notification_email: archana.m.shinde@intel.com
```
Comment (jodh-intel) [inline .travis.yml:22]: Until we find a way to allow team notifications in the config file, please could you add the team manually to https://scan.coverity.com/projects/clearcontainers-shim as maintainers/owners? That way, they'll get auto-notified after I run I believe. I've put in a manual request just now to get the ball rolling ;-)

**Comment by jodh-intel** [comment]
lgtm

File: .travis.yml (hunk 1 above)
Comment (amshinde) [reply .travis.yml:22]: Sure, approved yours, will add the rest

**Comment by jcvenegas** [comment]
lgtm

**Pull request merged**